app/src/main/java/com/x8bit/bitwarden/data/autofill/fido2/model/PasskeyGetCredentialsOptions.kt

```diff
@@ -0,0 +1,19 @@
+package com.x8bit.bitwarden.data.autofill.fido2.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Models a FIDO 2 credential selection request options received from a Relying Party (RP).
+ */
+@Serializable
+data class PasskeyGetCredentialsOptions(
```

Comment:
This looks like the same model as `PasskeyAssertionOptions`

Reply:
Good point. I wasn't sure if we should create separate classes since it's part of a different request. I double checked the spec and from what I can tell there are only two "option" entities. This is one not needed. Great catch!